xpublish/rest.py

```diff
@@ -222,11 +250,11 @@ def info():
             return self._info()
 
         @self._app.get('/dict')
-        def to_dict(data: bool = False):
-            return self._obj.to_dict(data=data)
+        def to_dict():
+            return self._obj.to_dict(data=False)
```

Comment:
This is a breaking change but I think it is needed for safety's sake. Exposing the full dataset as a single string could lead to bring down the server with a memory error.